public/css/main.css

```diff
@@ -318,9 +318,9 @@ template {
   --font-size: 1.6rem;
   --font-primary: Montserrat, sans-serif;
   --font-secondary: Raleway, sans-serif;
-  --container-small: 55rem;
-  --container-medium: 75rem;
-  --container-large: 100rem;
+  --container-small: 750px;
```

Comment:
Create a extra small container to keep the 550px breakpoint. Also change px back to em or rem?